invenio_accounts/utils.py

```diff
@@ -151,7 +151,7 @@ def default_confirmation_link_func(user):
     return token, _generate_token_url(endpoint, token)
 
 
-def register_user(_confirmation_link_func=None, **user_data):
+def register_user(_confirmation_link_func=None, send_register_msg=True, **user_data):
```

Comment:
I guess this is needed to be able to manually disable the email send? Related to invenio-oauthclient contrib login?